7.figures/nbconverted/Figure3_model_evaluation.r

```diff
@@ -139,12 +167,20 @@ f1_score_df <- readr::read_tsv(
         "Phenotypic_Class" = "c",
         "data_split" = "c",
         "shuffled" = "c",
-        "feature_type" = "c"
+        "feature_type" = "c",
+        "balance_type" = "c"
```

Comment:
Based on the plots, I am assuming that the balance type is no longer included, so could this be removed from the code since we have shifted to class-balanced models?

Reply:
On this specific command, I am loading a dataset using [`readr`](https://readr.tidyverse.org/). IMO, readr does even a better job than pandas at specifying data type, AKA `dtype`. This line that you are highlighting is reading in the column named "balance_type" (and other columns) as a character `dtype`. 

It is true that we only use balanced models in this visualization, but in order to do so, we need to make sure the column is loaded properly (specifically, as a character)